client/post-editor/editor-revisions/style.scss

```diff
@@ -1,17 +1,25 @@
-.editor-revisions {
-	color: $gray-text;
-	cursor: pointer;
-	display: block;
-	font-size: 13px;
-	margin: 0 0 -16px;
-	padding: 16px 0;
-	text-align: left;
-	width: calc(100% + 32px);
+/** @format */
+.editor-revisions__dialog {
+	padding: 0;
+}
+
+.editor-revisions__wrapper {
+	height: 100vh;
```

Comment:
We might want to revisit these dims (`100vh`/`100vw`) as they actually cause the modal area to extend off page (not visually though):

<img width="1783" alt="screen shot 2017-11-13 at 02 44 25" src="https://user-images.githubusercontent.com/4335450/32707565-b53fe994-c81c-11e7-8cb6-acf043be2c69.png">

Reply:
Good catch. Let's switch to % and make sure it computes against the modal size (`position: relative` on modal)

Reply:
I'll see if I can solve this quickly.  If not, going to move to a follow up PR

Reply:
This is kind of fidgety given the interaction w/ the Dialog styles.  Let's fix this in a follow up PR.